mosaic/build.gradle

```diff
@@ -60,10 +63,9 @@ allprojects {
 		}
 	}
 
-	tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile).configureEach { task ->
+	tasks.withType(KotlinJvmCompile).configureEach { task ->
```

Comment:
IntelliJ was showing yellow lines for these fully qualified names. I can undo these imports if you want.